docs/synapse/userguides/ug013_storm_ref_lift.rst

```diff
@@ -178,11 +178,17 @@ alltag()
 
 Lifts a set of nodes based on one or more tags.
 
+Optional parameters:
+
+* **Return limit:** specify the maximum number of nodes to be returned by the alltag query.
+
+  * ``limit=`` (operator syntax)
+
 **Operator Syntax:**
 
 .. parsed-literal::
   
-  **alltag(** *<tag>* [ **,** ...] **)**
+  **alltag(** *<tag>* [ **,** ...] **, limit=** *<limit>* **)**
```

Comment:
"limit" param is optional so should go inside the brackets:

`**alltag(** *<tag>* [ **,** *<tag>* **,** ... **, limit=** *<limit>* ] **)**`

Reply:
fixed